src/wavefront-obj-parser.js

```diff
@@ -2,47 +2,50 @@ module.exports = ParseWavefrontObj
 
 // Map .obj vertex info line names to our returned property names
 var vertexInfoNameMap = {v: 'vertex', vt: 'uv', vn: 'normal'}
-// The returned properties that we will populate
-var parsedProperties = ['normal', 'uv', 'vertex', 'normalIndex', 'uvIndex', 'vertexIndex']
+
```

Comment:
:clap:

btw, some additional info : replacing the `Number(indices[x])` with `parseInt(indices[x], 10)` improved the benchmark perf by about `200ms` (so about 10%). It was not really something I expected to be honest.